examples/Paxos/Paxos.qnt

```diff
@@ -1,298 +1,270 @@
 module Paxos {
```

Comment:
IDK why git got so lost here, but I just:
1. Added indentation
2. Change `MSGS` to use discriminated unions and got rid of the `TypeOK` check for it.
3. Fixed an import issue